08_cell_line_prediction/scripts/run_tcga_ccle_lasso.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Run feature selection experiments across a few genes, with selected cancer
+# types held out (either partially or completely)
+
+RESULTS_DIR=./08_cell_line_prediction/results/tcga_to_ccle_lasso_poc
+ERRORS_DIR=./ccle_lasso_poc_errors
+
+# number of features to "preselect" to by mean absolute deviation
+# -1 == no preselection
+MAD_PRESELECT=16042
+
+mkdir -p $ERRORS_DIR
+
+lasso_penalties=(
+  "0.001"
+  "0.005"
+  "0.01"
+  "0.05"
+  "0.1"
+  "0.5"
+  "1"
+  "5"
+  "10"
+  "50"
+  "100"
+  "250"
+  "500"
+  "750"
+  "1000"
+)
+
+genes=(
+  # "KRAS"
```

Comment:
Do you want to put these back in or leave them out?

Reply:
These I can put back in! Good catch.